scripts/templates/field_details.j2

```diff
@@ -112,14 +112,15 @@ Note also that the `{{ fieldset['name'] }}` fields are not expected to be used d
 
 [options="header"]
 |=====
-| Nested fields | Description
+| Nested fields | Location | Description
 
 // ===============================================================
 
 
 {% for entry in render_nestings_reuse_section -%}
 
 {#- Beta marker on nested fields -#}
+| `{{ entry['name'] }}`
```

Comment:
I wonder if it makes sense for the `Nested fields` column values to now contain the link to the field set vs. the reused location?

For example, looking at the `client.*` page from the [preview](https://ecs_1366.docs-preview.app.elstc.co/guide/en/ecs/master/ecs-client.html#_field_reuse_2), instead of `client.as.*`, `as` instead is linked to the `as.*` field set page.

Reply:
I think that would make a lot of sense! Will switch that over